client/src/utils/game-channel/game-channel.ts

```diff
@@ -54,6 +57,7 @@ export class GameChannel {
   timerStartTime = -1;
   lastOffChainTxTime = -1;
   channelIsClosing = false;
+  didReconnect = false;
```

Comment:
```suggestion
```

We can use `const didReconnect = !!localStorage.getItem('gameState');` to check if reconnection will be triggered and avoid increasing the total object fields of the game state